django_apscheduler/admin.py

```diff
@@ -39,6 +46,20 @@ def average_duration(self, obj):
 
     average_duration.short_description = "Average Duration (sec)"
 
+    actions = ['run_selected_jobs']
+
+    def run_selected_jobs(self, request, queryset):
+        scheduler = BackgroundScheduler()
+        scheduler.add_jobstore(DjangoJobStore(), "default")
+        scheduler.start()
```

Comment:
This will interfere with any other scheduler(s), that may already be monitoring the job store, and could cause the scheduled job executions to be missed and / or duplicated.

I understand that there is a relatively small risk of this happening given the short period of time that this `BackgroundScheduler` will be active for, but it is probably still not something that we can permit in production environments as the impact of this will vary from application to application (but could be severe).

Reply:
I'm understanding all of this, but from point of view "run now" for scheduled job is like an exception, for example: run at 7am failed, I see trace, see exception in Sentry, I'm creating fix, and I need rerun this job right after fix was deployed.